### PR TITLE
Updated samshee

### PR DESCRIPTION
**Pull request by nschcolnicov**
Addresses this:
https://github.com/nf-core/demultiplex/issues/237

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/demultiplex/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/demultiplex _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -233,5 +233,9 @@ process {
         ]
     }
```
Comment (apeltzer) [inline conf/modules.config:235]:
```suggestion
//Samshee should fail the entire pipeline immediately as it validated the illumina samplesheet to be valid before the pipeline runs. As such, it should not be running more than once & if it fails should stop the pipeline
```

**Review by apeltzer — APPROVED** [review]
Looks good to me - some explanation as a comment for config

File: conf/modules.config [hunk 2]
```diff
@@ -232,6 +232,10 @@ process {
             mode: params.publish_dir_mode
         ]
     }
+//Samshee should fail the entire pipeline immediately as it validated the illumina samplesheet to be valid before the pipeline runs. As such, it should not be running more than once & if it fails should stop the pipeline
```
Comment (edmundmiller) [inline conf/modules.config:235]:
```suggestion
// Samshee should fail the entire pipeline immediately as it validated the illumina samplesheet to be valid before the pipeline runs. As such, it should not be running more than once & if it fails should stop the pipeline
```

File: tests/pipeline/skip_tools.nf.test.snap [hunk 3]
```diff
@@ -174,9 +174,9 @@
             ]
         ],
         "meta": {
-            "nf-test": "0.9.0",
-            "nextflow": "23.10.1"
+            "nf-test": "0.8.4",
```
Comment (edmundmiller) [inline tests/pipeline/skip_tools.nf.test.snap:177]: Might want to bump your nf-test version to `0.9.0`

**Review by edmundmiller — APPROVED** [review]
Awesome, thanks for taking care of that so quickly!

**Pull request merged**